modules/ROOT/pages/fd-scatter-gather.adoc

```diff
@@ -0,0 +1,259 @@
+= Run Different Processes in Parallel, then Gather the Results
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+With a Scatter-Gather card, you can send a message through more than one set of cards at a time, collect the results from each set, and pass those results (along with the original message) to the next card in a flow.
+
+.This schematic diagram shows a Scatter-Gather card that includes the minimum of two routes.
+image::fd-sg.png[A Scatter-Gather card in a flow]
+<1> Any card can pass messages to a Scatter-Gather card, and this card can be the trigger of the current flow or any other card in the flow.
+<2> The Scatter-Gather card in included in a larger rectangle that is called the _scope_ of the card. Within this scope are two or more sets of card, each set called a _route_. The Scatter-Gather card processes all routes at the same time, and the routes are all independent of each other. Within a route, you can include any card, though it would not make sense to include an HTTP Listener because you don't want to hold up processing while waiting for an HTTP request. Each route can have one or more cards.
```

Comment:
```suggestion
<2> The Scatter-Gather card is included in a larger rectangle that is called the _scope_ of the card. Within this scope are two or more sets of a card, each set called a _route_. The Scatter-Gather card processes all routes at the same time and the routes are all independent of each other. Within a route, you can include any card, though it would not make sense to include an HTTP Listener because you don't want to hold up processing while waiting for an HTTP request. Each route can have one or more cards.
```

Reply:
I have to reinforce this: you are unable to add an HTTP Listener at this stage.